bindings/matlab/+iDynTreeWrappers/setWorldBaseTransform.m

```diff
@@ -0,0 +1,106 @@
+function [] = setWorldBaseTransform(KinDynModel,basePose)
+
+    % SETWORLDBASETRANSFORM sets the joints configuration for kino-dynamic 
+    %                       computations.
+    %
+    % This matlab function wraps a functionality of the iDyntree library.                     
+    % For further info see also: https://github.com/robotology/idyntree
+    %
+    % FORMAT: [] = setWorldBaseTransform(KinDynModel,basePose)
+    %
+    % INPUTS:  - KinDynModel: a structure containing the loaded model and additional info.
+    %          - basePose: [4 x 4] from base frame to world frame transform;
+    %
+    % Author : Lorenzo Moretti (lorenzo.moretti@iit.it)
+    %
+    % SPDX-FileCopyrightText: Fondazione Istituto Italiano di Tecnologia (IIT)
+% SPDX-License-Identifier: BSD-3-Clause
```

Comment:
```suggestion
    % SPDX-FileCopyrightText: Fondazione Istituto Italiano di Tecnologia (IIT)
    % SPDX-License-Identifier: BSD-3-Clause
```

Reply:
Makes totally sense. Done in [74d9e9d](https://github.com/robotology/idyntree/pull/1164/commits/74d9e9dccb3e6523cd64830e5a57196cf487f68d)

Reply:
My bad, committing now.